main.pm

```diff
@@ -498,7 +498,7 @@ sub load_consoletests() {
         loadtest "console/zypper_ref.pm";
         loadtest "console/yast2_lan.pm";
         loadtest "console/curl_https.pm";
-        if (!get_var("OFW")) {
+        if (!get_var("OFW") && !check_var("BACKEND", "s390x")) {
             loadtest "console/glibc_i686.pm";
```

Comment:
this is even worse as it should check_var('ARCH', 'x86_64') - it's pointless indepedent of the firmware or the backend used.